File: app/src/main/java/com/dlka/lastfmnewtracknotifier/Artist.java

package com.dlka.lastfmnewtracknotifier;

import com.squareup.moshi.Json;

public class Artist {

    @Json(name = "mbid")
    private String mbid;
    @Json(name = "#text")
    private String text;

    public String getMbid() {
        return mbid;
    }

    public void setMbid(String mbid) {
        this.mbid = mbid;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

}

File: app/src/main/java/com/dlka/lastfmnewtracknotifier/Artist.kt
package com.dlka.lastfmnewtracknotifier

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class Artist (
    @Json(name = "mbid")
    var mbid: String? = null,
    @Json(name = "#text")
    var text: String? = null
)
